apps/epl/src/epl.erl

```diff
@@ -23,24 +27,75 @@
 %% ===================================================================
 %% API functions
 %% ===================================================================
+
+%% Lookup for given Key in epl_priv ets
+-spec lookup(Key :: term()) -> [tuple()].
 lookup(Key) ->
     ets:lookup(epl_priv, Key).
 
+%% Add calling process to every epl_tracers' subscribers list
+-spec subscribe() -> [ok].
 subscribe() ->
-    epl_tracer:subscribe().
+    Nodes = get_all_nodes(),
+    [subscribe(N) || N <- Nodes].
+
+%% Add calling process to Node tracer's subscribers list
+-spec subscribe(default_node) -> ok;
```

Comment:
In the subscription functions you could use `node() | default_node` as spec for the `Node`.